controllers/allcontrollers.go

```diff
@@ -242,3 +257,19 @@ func Start(stopCh <-chan struct{}, client k8sclient.Interface) error {
 
 	return nil
 }
+
+// // getVolumeUsage gets the available and total capacity of a volume, in that order
```

Comment:
1. GetVolumeUsage instead of getVolumeUsage
2. var metricsPath instead of const metricsPath
3. Tests (with temp paths and defer cleanup of temp paths) in both the packages.

Reply:
`metricsPath` was originally suggested to be changed for `const` to `var` so that it can be changed for test cases but it does not seem be necessary.

I did choose to export it though.